bom/pom.xml

```diff
@@ -256,11 +256,6 @@ THE SOFTWARE.
         <artifactId>remoting</artifactId>
         <version>${remoting.version}</version>
       </dependency>
-      <dependency>
-        <groupId>org.jenkins-ci.modules</groupId>
-        <artifactId>instance-identity</artifactId>
```

Comment:
Once this is released, `bom-weekly` can get it.

Reply:
Though if I understand correctly from https://github.com/jenkinsci/bom/pull/671 https://github.com/jenkinsci/bom/pull/570#issuecomment-890346549 https://github.com/jenkinsci/bom/issues/681, it cannot be added until this PR is in the _oldest_ supported line, so that will be a while. In the meantime the handful of plugins using this dep can specify 3.1.